### DISCUSSION TITLE
How to regenerate content_html field

### DISCUSSION THREAD

**Discussion by comerc** [wintercms/wn-blog-plugin#62] [Q&A] [answered]
Tell me please.

I have done a database dump for the `content` field and would like to get a refresh for the `content_html` field.

I can go to edit a post in the admin panel and save the post, which will lead to HTML regeneration. But I have 175 posts. I would like to receive a command for batch execution.

**Comment by LukeTowers** [comment, marked as answer]
@comerc this would be something that Tinker would work well for:

```php
\Winter\Blog\Models\Post::all()->each(function ($post) { $post->save(); });
```
